sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md

```diff
@@ -14,7 +14,13 @@ For the best development experience, developers should use the official Microsof
 
 ### Install the package
 
-No package has been published for this client library yet.
+Install the Azure Time Series client Insights library for .NET with NuGet:
+
+```PowerShell
+Install-Package Azure.IoT.TimeSeriesInsights
+```
+
+View the package details at [nuget.org][nuget].
```

Comment:
do you intend to include the package url?

Reply:
Yes it's down below in readme.